frontend/tests/App.spec.js

```diff
@@ -59,6 +60,73 @@ describe('App.vue', () => {
     expect(wrapper.vm.collections).toEqual("collection");
   })
 
+  it('changeItemsState - method', () => {
+    wrapper = initComponent(App);
+    const fqid1 = "extName1.extPublisher1.id1";
+    const item1 = {
```

Comment:
Please provide clearer names for variables and id/fqid props.
`item1/2` `id1/2` `fqid1/2` are very generic names.

while something like: `topItem` and `nestedItem` are much clearer and tell the "story" of this test.